video/out/opengl/osd.c

```diff
@@ -44,9 +44,9 @@ struct vertex {
 };
 
 static const struct ra_renderpass_input vertex_vao[] = {
-    {"position",  RA_VARTYPE_FLOAT,      2, 1, offsetof(struct vertex, position)},
-    {"texcoord" , RA_VARTYPE_FLOAT,      2, 1, offsetof(struct vertex, texcoord)},
-    {"ass_color", RA_VARTYPE_BYTE_UNORM, 4, 1, offsetof(struct vertex, ass_color)},
+    {"position",  RA_VARTYPE_FLOAT,      2, 1, 0, offsetof(struct vertex, position)},
+    {"texcoord" , RA_VARTYPE_FLOAT,      2, 1, 1, offsetof(struct vertex, texcoord)},
+    {"ass_color", RA_VARTYPE_BYTE_UNORM, 4, 1, 2, offsetof(struct vertex, ass_color)},
```

Comment:
I disagree. This is 100% redundant.

Reply:
What would you do instead?

Reply:
I don't know the problem.

Reply:
Well, where would you generate the vertex locations? Have `gl_sc_set_vertex_format` make a copy of the array and mutate it? That sounds like annoying code to write.

Reply:
It's just the index?

Reply:
Not necessarily, it just happens to be the case for our (simple) vertex shaders. I guess we can hard-code that assumption into both shader_cache.c and ra_vk.c though, would that make you happier?

Reply:
I still don't understand the problem.